test/jdk/java/util/Locale/LanguageSubtagRegistryTest.java

```diff
@@ -24,9 +24,9 @@
 /*
  * @test
  * @bug 8040211 8191404 8203872 8222980 8225435 8241082 8242010 8247432
- *      8258795 8267038
+ *      8258795 8267038 8287180 8302512
  * @summary Checks the IANA language subtag registry data update
- *          (LSR Revision: 2022-08-08) with Locale and Locale.LanguageRange
+ *          (LSR Revision: 2023-2-14) with Locale and Locale.LanguageRange
```

Comment:
Add prepending `0` to the month to keep the YYYY-MM-DD format.

Reply:
Thanks for catching, fixed